src/modules/rc_update/params.c

```diff
@@ -1329,6 +1329,38 @@ PARAM_DEFINE_INT32(RC_MAP_YAW, 0);
  */
 PARAM_DEFINE_INT32(RC_MAP_FLTMODE, 0);
 
+/**
+ * Single channel R/C enable switch
+ *
+ * If this parameter is non-zero and this channel is asserted,
+ * then R/C controls are not passed to the Autopilot
+ *
```

Comment:
I'm not sure someone who didn't know about this could infer what it was for. How about 

```suggestion
 * Single channel RC-Input-Enable switch
 *
 * Set an RC channel for enabling/disabling other all other RC input to the autopilot.
 * The threshold for switch activation is set using RC_ENABLESW_TH.
 *
```